.github/workflows/continuous-integration.yml

```diff
@@ -156,3 +206,337 @@ jobs:
 
       - name: "Upload Code Coverage"
         uses: "codecov/codecov-action@v1"
+
+  phpunit-postgres:
+    name: "PHPUnit with PostgreSQL"
+    runs-on: "ubuntu-latest"
+    needs: phpunit-sqlite
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+        postgres-version:
+          - "9.2"
+          - "9.3"
+          - "9.4"
+          - "9.5"
+          - "9.6"
+          - "10"
+          - "11"
+          - "12"
+          - "13"
+
+    services:
+      postgres:
+        image: postgres:${{ matrix.postgres-version }}
+        env:
+          POSTGRES_PASSWORD: postgres
+
+        options: >-
+          --health-cmd "pg_isready || psql -U postgres -c 'SELECT 1'"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
Where do these numbers come from? I don't know what the defaults are but somehow we got away with not specifying anything like that for the Oracle builds, and they work. At the same time, some similar options seem to cause issues in https://github.com/doctrine/doctrine-website/pull/367.

Reply:
They come from https://docs.github.com/en/free-pro-team@latest/actions/guides/creating-postgresql-service-containers, I will try removing them.